src/Wrap.js

```diff
@@ -37,23 +38,23 @@ const Wrap = (props: WrapProps): React.Element<*> => {
             <animate
```

Comment:
My initial idea is removing all elements tag `<animate />` and not just set animation as zero

Reply:
I second that. When animation is set to false, then any element representing animation should be removed from DOM, if possible.  There should be no unused elements.

Reply:
Makes sense! Changed this section of code too

Reply:
👍